packages/flutter/test/cupertino/bottom_tab_bar_test.dart

```diff
@@ -327,6 +327,48 @@ Future<void> main() async {
     expect(tester.getSize(find.byType(CupertinoTabBar)).height, 90.0);
   });
 
+  testWidgets('Set height', (WidgetTester tester) async {
+    const double tabBarHeight = 56.0;
```

Comment:
We usually add references this this for future curious minds. :)
```suggestion
    // Regression test for https://github.com/flutter/flutter/issues/51704
    const double tabBarHeight = 56.0;
```